lib/components/map/point-popup.tsx

```diff
@@ -50,6 +50,65 @@ function MapPopup({
   const intl = useIntl()
   const { current: map } = useMap()
 
+  /**
+   * Creates a focus trap within map popup that can be dismissed with esc.
+   * https://medium.com/cstech/achieving-focus-trapping-in-a-react-modal-component-3f28f596f35b
+   */
+  function getEntryRelativeTo(
```

Comment:
Can this function be refactored for the different places a similar functionality is used?